app/views/categories/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="navigation">
+  <ul>
+    <li><%= link_to image_tag('backarrow.png', alt: 'Back'), categories_path, class: 'nav-link' %></li>
+    <h2>Create A Category</h2>
+    <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" %></li>
+  </ul>
+</div>
+<% if @category.errors.any? %>
+  <div id="error_explanation">
+    <h2><%= pluralize(@category.errors.count, "error") %> prohibited this category from being saved:</h2>
+    <ul>
+    <% @category.errors.full_messages.each do |message| %>
+      <li><%= message %></li>
+    <% end %>
+    </ul>
+  </div>
+<% end %>
+<div class="form-class">
+<%= form_with(model: @category, url: categories_path, method: :post, multipart: true) do |form| %>
+    <%= form.text_field :name, placeholder: "Enter your name" %>
+    <%= form.file_field :icon, id: 'icon_upload' %>
+    <%= form.submit "Create Category" %>
+<% end %>
```

Comment:
- [ ] Kindly design the **_form page_** for adding a new **_category_** to look identical to that of the design layout provided for forms. In the design layout, a form page for registering a user was provided. Kindly use the design layout for other form pages in your application. Designing a page based on a wireframe or sample layout ensures consistent and accurate implementation of the intended user experience, reducing ambiguity and misinterpretation. 🖱️ 🖥️ 


| Your Design  | Behance Design |
| ------------- | ------------- |
| ![image](https://github.com/misalazeem/Budget-App/assets/77106962/170e50e2-c171-4f0b-a4db-e022c0721734) | ![image](https://github.com/yegonkimutai/budget-app/assets/77106962/1167219c-4a3b-48e4-b19d-22ebb21c5edd) |

#### Hint: Please pay attention to the design of the **_navigation bar and the form inputs_**. 😎